### PR TITLE
Http2 fix

### PR DESCRIPTION
**Pull request by spisiakm**
Fixes `testHttp2_Internal_H2C` test and updates some imports in `sockjs-it` to conform with Vert.x 3.5.1 structure

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+language: java
```
Comment (cescoffier) [inline .travis.yml]: why do we have a .travis file?

File: http2-it/src/test/java/io/vertx/openshift/http2/Http2IT.java [hunk 2]
```diff
@@ -85,6 +86,7 @@ public void testHttp2_H2() throws Exception {
    */
   @Test
   public void testHttp2_Internal_H2C() throws Exception {
+    System.out.println("Running internal H2C test - HTTP/2...");
```
Comment (cescoffier) [inline http2-it/src/test/java/io/vertx/openshift/http2/Http2IT.java]: trace can be removed

File: http2-it/src/test/java/io/vertx/openshift/http2/Http2IT.java [hunk 3]
```diff
@@ -51,6 +51,7 @@ public static void initialize() throws IOException {
    */
   @Test
   public void testHttp2_H2() throws Exception {
+    System.out.println("Running a TLS passthrough HTTP/2 test");
```
Comment (cescoffier) [inline http2-it/src/test/java/io/vertx/openshift/http2/Http2IT.java]: trace can be removed

File: http2-it/src/test/java/io/vertx/openshift/http2/Http2IT.java [hunk 4]
```diff
@@ -106,7 +108,9 @@ public void testHttp2_Internal_H2C() throws Exception {
     await().atMost(1, TimeUnit.MINUTES).untilAtomic(response, is(notNullValue()));
 
     assertThat(response.get())
-      .startsWith("Aloha HTTP_2");
+      .contains("version = HTTP_2")
+      .contains("Aloha from vert.x!")
+      .contains("HTTP_1_1");
```
Comment (cescoffier) [inline http2-it/src/test/java/io/vertx/openshift/http2/Http2IT.java]: how come it does contain HTTP_1_1 and HTTP_2 ?

File: http2-it/src/test/java/io/vertx/openshift/http2/Http2IT.java [hunk 5]
```diff
@@ -143,6 +147,7 @@ public void testHttp2_H2C() throws Exception {
 
   @Test
   public void testGRPC() throws Exception {
+    System.out.println("Running GRPC test...");
```
Comment (cescoffier) [inline http2-it/src/test/java/io/vertx/openshift/http2/Http2IT.java]: Remove trace

File: .travis.yml (hunk 1 above)
Comment (spisiakm) [reply .travis.yml]: Forgot to remove it, I was just playing with Travis.

File: http2-it/src/test/java/io/vertx/openshift/http2/Http2IT.java (hunk 4 above)
Comment (spisiakm) [reply http2-it/src/test/java/io/vertx/openshift/http2/Http2IT.java]: HTTP_1_1 is used to communicate with `EdgeVerticle`, which then communicates with `HelloHttp2Verticle` using HTTP_2. The response body from `HelloHttp2Verticle` contains something like
```html
<html>
  <body>
    <h1>Aloha from vert.x!</h1>
    <p>version = HTTP_2</p>
  </body>
</html>
```
This response is received by `EdgeVerticle`, which then appends the http version of the request from `Http2IT`, in our case `HTTP_1_1`. So I'm testing that the response contains both.

**Pull request merged**